app/models/alchemy/eager_loading.rb

```diff
@@ -25,6 +25,7 @@ def page_includes(version: :public_version)
               elements: [
                 :page,
                 :touchable_pages,
+                :tags,
```

Comment:
Does this need to be `taggings: :tag`? Or is there a `has_many :tags, through: :taggings` relation?

Reply:
There's a `has_many through` defined, and we access the tag list through `:tags`. Preloading `taggings: :tags` works, but will not preload the association we actually use. Compare also the tags preload on the page that works the same way.

Reply:
Thanks